### DISCUSSION TITLE
Timesheet uncheck billable by default

### DISCUSSION THREAD

**Discussion by cfshawnmaholick** [kimai/kimai#2624] [Questions & Answers]
![Screenshot 2021-06-25 at 15 44 16](https://user-images.githubusercontent.com/36766765/123434172-7bae5780-d5cc-11eb-85e4-16146890d363.png)
Hello, is there a way to configure the billable checkbox to not be checked by default?

Beside my main question:
Is there a way by config to remove optional fields from the form?

Thanks,
Shawn

**Comment by kevinpapst** [comment]
No, but the billable feature will be improved for the next version.

**Reply by cfshawnmaholick** [reply]
Hi Kevin, thanks alot for your answer. That's awesome!
